meta-ibm/recipes-kernel/linux/linux-aspeed/p10bmc.cfg

```diff
@@ -1,3 +1,7 @@
+CONFIG_AUDIT=y
+CONFIG_AUDITSYSCALL=y
```

Comment:
also do we need to enable the systemically audit as well ? we are not auditing the system calls right ? do we think enabling this would have any performance impact ? even though we don't use it ?